src/documentation_server.ml

```diff
@@ -0,0 +1,40 @@
+open Import
+
+type t = Polka.Server.t
+
+let start ~path ?(port = 0) () =
```

Comment:
Since we are never passing the `?port` parameter, I would just get rid of it and hardcode 0. No use in making things general here.